pkg/containerd/events.go

```diff
@@ -97,6 +112,20 @@ func (e *EventMonitor) MonitorPorts(ctx context.Context) {
 					continue
 				}
 
+				// We only need to perform this check
+				// when the namespaced network is enabled
+				// in the case of default network when the
+				// privileged service is disabled, we will
+				// not be using this event API.
+				if !e.enablePrivilegedService {
+					if err = execIptablesRules(
+						ports, startTask.ContainerID,
+						envelope.Namespace,
+						strconv.Itoa(int(startTask.Pid))); err != nil {
+						log.Errorf("failed running iptable rules to update DNAT rule in CNI-HOSTPORT-DNAT chain: %v", err)
```

Comment:
This indenting is unreadble.

Please split this to two lines (`err = execIptablesRules(…); if err != nil {`) due to line length reasons.

(I was going to say "move the brace to start of line to make it more readable", but splitting into two lines is clearer.)

Please also file an issue for a refactor (to make the contents of `case "/tasks/start":` a new helper method). I assume that's doable, at least; if it looks unreasonable we can skip that (though it's not great).